.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,16 @@
+- id: gdlint
+  name: gdlint
+  description: "gdlint - linter for GDScript"
+  entry: gdlint
+  language: python
+  language_version: python3.7
```

Comment:
Why do require 3.7, and not let the default Python installation be run? 
If a User does have an incompatible default Version, he still can adjust it in his `.pre-commit-config.yaml`

Reply:
@MinerSebas AFAICT godot-gdscript-toolkit requires python 3.7 or greater.
Since this PR https://github.com/Scony/godot-gdscript-toolkit/pull/8 Python 3.6 isn't supported anymore and is not available in pypi.

It can still be overridden in the `.pre-commit-config.yaml`.
I have tested it with python 3.8 and python 3.7 in the `.pre-commit-config.yaml` and it worked.

I think it can help people to not try using a non compatible version.

BTW thanks for linking it to godot-ci-template sounds interesting :+1:

Reply:
I can change to `python3` and add a comment about the compat if you think it's better.
I didn't find a way with `pre-commit` to set a minimum version for python.